app/controllers/foreman_tasks/react_controller.rb

```diff
@@ -0,0 +1,11 @@
+module ForemanTasks
+  class ReactController < ::ApplicationController
```

Comment:
why is that needed?

Reply:
We want to implement it with `react-router`

Reply:
ack, lets finalize the way its supposed to be used in foreman first? ( i would assume as a plugin you would reuse that infra)

Reply:
agree, imo foreman should provide `registerRoutes` so we won't need to use turbolinks when moving between foreman-routes to plugin-routes.

Reply:
@sharvit how much work would it be to do the above?